contents/go/Go.md

```diff
@@ -0,0 +1,143 @@
+# Introduction to Go
+
+Author(s): [Cara Leong](https://github.com/craaaa)
+
+Go (also known as `golang`) is a compiled, statically-typed, garbage-collected language that has special memory safety and concurrent programming features. Born out of frustration with the available languages (e.g. C, C++, Java) and environments for systems programming, Go was [conceptualized by programmers at Google](https://talks.golang.org/2012/splash.article) who sought to create a single language that was efficient to write, build and execute. Go also supports newer developments in computing such as multicore processors and network systems.
+
+Go is an [open source project](https://github.com/golang/go). Its source code may be useful reading for those interested in learning good practices, or simply to find out more about how the language was implemented.
+
+## Getting started
+
+Go provides its own [installation guide](https://golang.org/doc/install) and an interactive [ tour of Go](https://tour.golang.org/). These are useful and highly comprehensive resources for programmers looking to learn the syntax and style of Go. For those who prefer to read existing code examples, [Go by Example](https://gobyexample.com/) is a collection of code samples covering a wide variety of features in Go, and includes line-by-line explanations of the code. For those looking for a quick crash course on Go syntax, the Learn X In Y Minutes [Go cheatsheet](https://learnxinyminutes.com/docs/go/) may also be a good starting point.
+
+If you're unconvinced about Go, [the Go playground](https://play.golang.org/) can be used as a sandbox in which to write, build and execute code without installing Go on your machine.
+
+## Using Go
```

Comment:
Add a section on variable declaration, e.g. the difference between using `var`, `:=` for inference, etc. You can also cover the reversed adjective order, e.g. in Java we specify `String message` but in Go we say `message string`. https://blog.golang.org/gos-declaration-syntax

You can also cover pointers in the same section, since they're a thing in Go but their declaration syntax is also different.